Additional translations (part 2)

noccu opened the pull request:
Now with more editing and testing.
Uma profile blurbs could use a proofread as I'm not too familiar with most of them yet.

Also I'm not sure what the basis is for adding categories to the import script as I noticed not all of the csv files are listed there?

noccu committed:
Dump and translate more categories

Gacha info & names, presents info, mission groups,
Uma profiles catchcopies & weight.

"Commemorative Missions" takes up a whole line by itself
so I just dropped "Commemorative" as not overly important.

noccu committed:
Add piece items translation.

FabulousCupcake commented:
> Also I'm not sure what the basis is for adding categories to the import script as I noticed not all of the csv files are listed there?

The ones that are listed in the import script should be or is more frequently updated; but you got a point — why not put them all in there? 🤔

noccu committed:
Add campaigns translation.

Forgot to include this as it's used in presents-desc.csv
and was already dumped.
Also fix some small errors and add every csv
in this branch to the import script.
